extensions/binding/org.eclipse.smarthome.binding.fsinternetradio/META-INF/MANIFEST.MF

```diff
@@ -7,20 +7,22 @@ Bundle-Version: 0.9.0.qualifier
 Bundle-RequiredExecutionEnvironment: JavaSE-1.7
 Bundle-ClassPath: .
 Import-Package: 
- org.apache.commons.httpclient,
- org.apache.commons.httpclient.methods,
- org.apache.commons.httpclient.params,
  org.apache.commons.io,
+ org.eclipse.jetty.client,
+ org.eclipse.jetty.client.api,
+ org.eclipse.jetty.http,
+ org.eclipse.jetty.util.component,
  org.eclipse.smarthome.config.core,
+ org.eclipse.smarthome.config.discovery,
  org.eclipse.smarthome.core.library.types,
  org.eclipse.smarthome.core.thing,
  org.eclipse.smarthome.core.thing.binding,
  org.eclipse.smarthome.core.thing.binding.builder,
  org.eclipse.smarthome.core.thing.type,
  org.eclipse.smarthome.core.types,
- org.eclipse.smarthome.config.discovery,
  org.jupnp.model.meta,
- org.slf4j
+ org.slf4j,
+ org.eclipse.smarthome.io.net.http
```

Comment:
Can you keep the alphabetical order (for manifest first it is perhaps easier to manage if it is ordered).